src/Module/Photo.php

```diff
@@ -82,27 +82,47 @@ public static function rawContent(array $parameters = [])
 				$square_resize = !in_array($parameters['type'], ['media', 'preview']);
 			}
 
+			if (!empty($parameters['guid'])) {
+				$guid = $parameters['guid'];
+				$account = DBA::selectFirst('account-user-view', ['id'], ['guid' => $guid], ['order' => ['uid' => true]]);
+				if (empty($account)) {
+					throw new HTTPException\NotFoundException();
+				}
+
+				$id = $account['id'];
+			}
+
+			// Contact Id Fallback, to remove after version 2021.12
+			if (!empty($parameters['contact_id'])) {
```

Comment:
Forgot about the `isset()`.
```suggestion
			if (isset($parameters['contact_id'])) {
```